### ISSUE TITLE
Control Points

### ISSUE THREAD

**Issue by aartykov** [CVMI-Lab/SC-GS#51]
Hello! Thanks for the great project. My question is how to get the resultant control points after training? I want to visualize them on the canonical space.

**Comment by yihua7** [comment]
Hi,
Thanks for your interest!
- You can visualize the control points by clicking the `node` button on the control panel when using the GUI window.

- You can also refer to the definition of control points **(in the canonical space)** at https://github.com/yihua7/SC-GS/blob/26cd57d09598b2f5d951029808a5ac9f0ff4f626/utils/time_utils.py#L806 and export them as point cloud by writing some codes (e.g `import trimesh;trimesh.Trimesh(verts=nodes[..., :3].detach().cpu().numpy()).export('control_points.ply')`). Please remember to load the checkpoint before doing that. : )
